examples/hello/client.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) Bull S.A.S - 2024
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program; if
+ * not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ *
+ * BXI Low Level Team
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "portals4.h"
+#include "portals4_bxiext.h"
+
+#include "hello.h"
+
+/*
+ * this file represent a client sendind a message to a server
+ *  
+ * Usage: ./client [nid] [message]

```

Comment:
```suggestion
 * Usage: ./client <nid> <message>
```

The difference between `[argument]` and `<argument>` is that the first one is optional, whereas the second one is mandatory